### ISSUE TITLE
POS does not appear to tally fractional quantities

### ISSUE THREAD

**Issue by JBruceDawson** [Dolibarr/dolibarr#8170]
# Bug
POS does not appear to tally fractional quantities. If a fractional quantity such as (1.15) is entered, then the total is zero!

## Environment
- **Version**: 6.0.5
- **OS**: Linux
- **Web server**: Apache/2.4.25 (Debian)
- **PHP**: apache2handler 7.0.27-0+deb9u1
- **Database**: MySQL or MariaDB 5.5.5-10.1.26-MariaDB-0+deb9u1
- **URL**: /dolibarr/comm/card.php?socid=7

## Report
![screenshot - 02092018 - 03 10 11 pm](https://user-images.githubusercontent.com/11779528/36047897-72ae95b0-0dab-11e8-9fa7-46b0a43bc179.png)

**Comment by eldy** [comment]
Seems ok with 7.0

**Comment by JBruceDawson** [comment]
Just upgraded to 7.0.0 on Debian Stretch, and this is what I'm seeing...
So I think it is still broken.

The first entry was with 6.0.5, and the second entry was after upgrading
to 7.0.0 (which went very smoothly)!

--Bruce

PS: Note that this installation started with Dolibarr 4.0.5 from
Debian's distribution. Then I used dolibarr_6.0.5-4_all.deb to upgrade
to 6.0.5 in an attempt to fix the problem with PHPSpreadsheet, and it
turns out that required a config file fix. Now I have upgraded using
dolibarr_7.0.0-4_all.deb in an attempt to fix the fraction quantities
problem in POS.
